.github/workflows/nowsecure.yml

```
@@ -0,0 +1,8 @@
+name: NowSecure Scan
+on: [push, pull_request]
+jobs:
```

actionlint (1.7.4)

"jobs" section should not be empty. please remove this section if it's unnecessary

(syntax-check)